src/senaite/core/api/dtime.py

```diff
@@ -436,3 +437,39 @@ def to_localized_time(dt, long_format=None, time_only=None,
                 formatstring = "[INTERNAL ERROR]"
         time_str = date_to_string(dt, formatstring, default=default)
     return time_str
+
+
+def get_relative_delta(from_dtime, to_dtime=None):
+    """Returns the relative delta between two datetimes. If to_dtime is None,
+    compares current datetime with from_dtime
+    """
+    if not to_dtime:
+        to_dtime = datetime.now()
+
+    from_dtime = to_dt(from_dtime)
+    to_dtime = to_dt(to_dtime)
+    if not all([from_dtime, to_dtime]):
+        return None
```

Comment:
Better raise an Exception if the parameters can not be converted to valid dates.

Reply:
Done with ec737946df1dd8b1e809a44f49b25770e0c7632e